src/main/java/com/michalsydoryk/app/points/Points.java

```diff
@@ -0,0 +1,26 @@
+package com.michalsydoryk.app.points;
+
+import java.math.BigDecimal;
+
+public class Points {
+
+    private static final int INITIAL_VALUE = 0;
```

Comment:
Why constant?

Reply:
Why not?

Reply:
For me static fields are used when they represent something universal also outside of the class. If its used only inside in this particular class there is no need to take memory place needlesly.